CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - feat(fluent-bit): drop all capabilities for container [#2151][#2151]
 - feat: allow to collect logs from /var/log/pods and add instruction how to do it [#2153][#2153] [#2156][#2156]
 - feat(otellogs): support tolerations, nodeSelector and affinity for daemonset [#2158][#2158]
+- chore(fluent-bit): update to 0.14.1 [#2155][#2155]
```

Comment:
Could you mention Fluent Bit version that is used in this version of Helm chart in the changelog? I  see that it is Fluent Bit 1.7.2

Reply:
We override it in our `values.yaml` and I would stick to it